client/src/components/MeetingExperience.test.tsx

```diff
@@ -214,21 +226,15 @@ describe('MeetingExperience', () => {
         waitingSubtitle={waitingSubtitle}
         logoUrl={logoUrl}
         chatEnabled={true}
-        postCall={undefined}
+        postCall={mockPostCall}
         onDisplayError={jest.fn()}
       />
     );
 
     await runFakeTimers();
-
     meetingExperience.update();
 
-    expect(setRenderPostCallMock).toBeCalled();
     const survey = meetingExperience.find(Survey);
-    expect(survey.length).toBe(0);
-    const callWithChatComposites = meetingExperience.find(CallWithChatComposite);
-    expect(callWithChatComposites.length).toBe(1);
-    const parentDiv = callWithChatComposites.parent();
-    expect(parentDiv.props().style.display).toBe('flex');
+    expect(survey.length).toBe(1);
```

Comment:
Let's bring back the assertions on CallWithChatComposite to truly test that only Survey is visible:

```
    const callWithChatComposites = meetingExperience.find(CallWithChatComposite);
    expect(callWithChatComposites.length).toBe(1);
    const parentDiv = callWithChatComposites.parent();
    expect(parentDiv.props().style.display).toBe('none');
```

Reply:
done